src/Pretzel.Logic/Extensions/YamlExtensions.cs

```diff
@@ -107,7 +109,11 @@ public static string ExcludeHeader(this string text)
             if (m.Count == 0)
                 return text;
 
-            return text.Replace(m[0].Groups[0].Value, "").Trim();
+            text = text.Replace(m[0].Groups[0].Value, "");
+            var lines = text.Split(new[] { "\r\n", "\n" }, StringSplitOptions.None).ToList();
+            while (lines.Any() && string.IsNullOrWhiteSpace(lines.First()))
```

Comment:
You can't remove empty line in the middle of the text. My `while` removes only first empty lines. So, `SkipWhile` is needed. I will fix it.